Description: make check for commented out values really only check ones
  which are commented out
Author: Jamie Strandboge <jamie@ubuntu.com>
Forwarded: no

Index: libvirt-0.8.7/tests/daemon-conf
===================================================================
--- libvirt-0.8.7.orig/tests/daemon-conf	2010-12-20 07:35:22.000000000 -0600
+++ libvirt-0.8.7/tests/daemon-conf	2011-02-08 12:35:39.473066462 -0600
@@ -19,7 +19,7 @@
 conf="$abs_top_srcdir/daemon/libvirtd.conf"
 
 # Ensure that each commented out PARAMETER = VALUE line has the expected form.
-grep -v '\"PARAMETER = VALUE\"' "$conf" | grep '[a-z_]  *=  *[^ ]' | grep -vE '^#[a-z_]+ = ' \
+grep -v '\"PARAMETER = VALUE\"' "$conf" | grep '^#' | grep '[a-z_]  *=  *[^ ]' | grep -vE '^#[a-z_]+ = ' \
   && { echo "$0: found unexpected lines (above) in $conf" 1>&2; exit 1; }
 
 # Start with the sample libvirtd.conf file, uncommenting all real directives.
